index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
+
+	When a [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>. 
+
+	As a specification progresses along the Recommendation Track, Working Groups
+	<em class="rfc2119">should</em> solicit review from some or all of the following
+	stakeholders: 
+
+	<ul>
+		<li>The Working Group responsible for the specification</li>
+		<li>The horizontal review groups (see 6.2.3.2)</li>
+		<li>The Working Groups and organisations documented in the Working Group's charter</li>
+		<li>The general public</li>
+		<li>Other relevant standards organizations</li>
+	</ul>
+
+	The Director <em class="rfc2119">must</em> consider evidence of wide review before
+	approving transitions. Appropriate evidence may include: 
+
+	<ul>
+		<li>Documented requests for wide review</li>
+		<li>Documented discussion and resolution of issues raised during wide review</li>
+	</ul>
+
+<h5 id="horizontal-review">
+Horizontal Review</h5>
+
+	The objective of <dfn id=dfn-horizontal-review export>horizontal review</dfn> is
+	to make sure specifications are robust in terms of accessibility, internationalisation,
+	privacy, security, and technical architecture by the time they become a [=W3C Recommendation=]. 
+
+	Horizontal review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, so that accessibility, internationalisation, privacy,
+	security, and technical architecture are considered throughout. 
+
+	Before a proposed specification transitions onto the Recommendation Track, the
+	group responsible for the specification <em class="rfc2119">must</em> solicit a
+	horizontal review from each of the horizontal review groups (in many cases,
+	submitting a self-review to the reviewing group will satisfy this requirement).
+
+	The <dfn id=dfn-horizontal-review-groups export>Horizontal Review Groups</dfn> are:
+
+	<ul>
+		<li><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a></li>
+		<li><a href="https://www.w3.org/International/core/Overview">Internationalisation Working Group</a></li>
+		<li><a href="https://www.w3.org/Privacy/">Privacy Interest Group</a></li>
+		<li><a href="https://www.w3.org/2001/tag/">Technical Architecture Group</a></li>
+	</ul>
+
+	When the [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to
+	<a href="https://lists.w3.org/Archives/Public/horizontal-review-announce/">horizontal-review-announce@w3.org</a>. 
+
+	Horizontal review <em class="rfc2119">must</em> be solicited from the Horizontal
+	Review groups at least 60 days before requesting transition to [=Candidate Recommendation=]. 
```

Comment:
I think this needs to be re-phrased; you cannot ask for a CR transition any sooner that 60 days after the latest HR group was made aware of the specification and the opportunity to comment.

Reply:
whoops, you can only ask for a CR transition if either (a) horizontal groups have been given at least 60 days or (b) they have completed their reviews. (To complete the transition, you also need to address the issues raised, of course).

The way I phrased it before put a mandatory 60-day delay, which is silly if the HR groups have completed their review earlier.

Reply:
I don't think either Léonie's text or David's quite work.

One (mis)interpretation of the text is that the HR solicitation sent at FPWD counts, and thus this boils down to CR must not be sooner than 60 days after FPWD. I doubt that's the intent. Similarly, if you get another round of HR, then change everything in the spec, and try to go to CR, it shouldn't work, regardless of how many days ago it was, since the review is now outdated. If earlier reviews don't count, to send a request for review 60days before CR without delaying CR by 60 days, you need to know when CR is, and not change anything of significance between requesting the review and going for CR. In the general case at least, that date isn't known, since the CR entrance criteria are quality driven, not date driven. As long as you have blocking issues, you cannot enter CR and you generally don't know how long it will take to solve them (nor whether additional blocking issues will be found in the meanwhile).

I think we should instead go with something like:
> In addition to continuous cooperation with HR groups and to the notification sent to public-review-announce@w3.org upon entering FPWD, Working Groups should explicitly notify HR groups when when major changes occur in a specification which has already been reviewed.
> Note: failure to do so early may cause delays in publishing a CR: reviews take time, and having received Wide Review, which includes Horizontal Review, is a mandatory criterion for publishing a CR. Allowing for 60 days or more is recommended.

Reply:
I like this wording @frivoal and note that there's a key point here around which the precise wording hinges: 
1. are HR groups being given a maximum of 60 days to make some sort of response, otherwise WGs can proceed? or 
2. are WGs being required to give evidence of completed HR regardless of how long it takes?

I got the sense before that the intent is 1, which I'd be happy with as a Chair, because it lets my group make progress, but I think 2 would also be reasonable as long as we don't get a situation where WGs are blocked waiting for HRs to be completed by HR groups that are completely snowed under. In this latter situation, I think we need some pragmatism to allow something to give.

Reply:
This is OK. I don't like the word "solicit", as previously noted. And I wanted to give groups the ability to progress if the HR groups don't respond; i.e. you can progress if they have responded and you dealt with it, or you've given them at least 60 days.